examples/exoskeleton/index.html

```diff
@@ -51,6 +51,8 @@
 		</script>
 		<script src="node_modules/todomvc-common/base.js"></script>
 		<script src="node_modules/exoskeleton/exoskeleton.js"></script>
+		<script src="node_modules/backbone.nativeview/backbone.nativeview.js"></script>
+		<script type="text/javascript">Backbone.View = Backbone.NativeView;</script>
```

Comment:
Can this be done in a script instead or is this a requirement for the dependencies that follow? In that case, could you remove the `type` attribute?